src/CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ add_library(StylePlugin MODULE
 )
 target_link_libraries(StylePlugin StyleSheetParser Qt5::Quick)
 
+set_target_properties(StylePlugin PROPERTIES PREFIX "")
```

Comment:
Travis CI is failing with errors like
```
file:///Users/travis/build/Ableton/aqt-stylesheets/tests/test-basic.qml:7:1: module "Aqt.StyleSheets" plugin "StylePlugin" not found 
     import Aqt.StyleSheets 1.1 
     ^
```
because on macOS, the plugin is expected to start with `lib`.
Please amend these changes so they are only applied on Windows (i.e. `if(WIN32)`).